.golangci.yaml

```diff
@@ -27,8 +27,6 @@ linters:
   - nolintlint
   - prealloc
   - revive
-  - staticcheck
-  - stylecheck
```

Comment:
Why turn off the check, it works fine.

Reply:
i didnt turn off the check, static check and style check are deprecated and it's covered under the global run.go in the later versions, PFA the image where the warnings are shown
![image](https://github.com/kubernetes-sigs/kwok/assets/109957201/05af1a7c-8b0a-410e-b347-92e6057000ab)